python/cudaq/kernel/ast_bridge.py

```diff
@@ -561,7 +561,7 @@ def bodyBuilder(iterVar):
                                       rawIndex).result
             loadedEle = cc.LoadOp(eleAddr).result
             castedEle = self.promoteOperandType(targetEleType, loadedEle)
-            targetEleAddr = cc.ComputePtrOp(targetElePtrType, targetPtr,
+            targetEleAddr = cc.ComputePtrOp(targetElePtrTy, targetPtr,
```

Comment:
Changes in this file look like simple respellings of identifier and NFC. Could be moved to a separate PR.

Reply:
Ah it was a bug that sneaked into main and was fixed separately here in another PR, and then merged:) Will undo

Reply:
done